.travis.yml

```diff
@@ -15,6 +15,7 @@ before_install:
   - 'nvm install --lts'
   - 'export PATH=./node_modules/.bin:$PATH'
   - 'npm install -g typescript'
+  - 'sudo apt-get -qq -y install conntrack'
```

Comment:
What is conntrack used for?

Reply:
Looks like it's necessary for minikube 1.18 now based on some github issues -- https://github.com/kubernetes/minikube/issues/6619 and https://github.com/kubernetes/minikube/issues/7179 for example

Reply:
Ah, thanks for clarification.